### DISCUSSION TITLE
Evolving the profile and some questions

### DISCUSSION THREAD

**Discussion by deadeyeflint** [suchmememanyskill/PrusaSlicer-Ender3-v3-SE-Config#13] [General]
Hey buddy, 

I really appreciate ur work! I've got my E3V3SE a month ago and started my profile from scratch, till i found urs here on github. your profile works quite better then mine :D but i'm not working with ur start and end gcode. i did a fork of my old settings and ur profiles.

there are a few question i got, while i was working with your profile (till now only the slow one):

1. the slow profile works quite fine, for my pla+ (jayo) filaments, but stringing is a big issue, as u mentioned in ur release notes. Which type of filament are u using?
2. Because of the mentioned issue with stringing and rounded edges, maybe the filament is not made for that high speeds? there are special filaments for speeds over ~120mm/s with less stringing, better layer adhesion, etc. 
3. because of the round edges: do u tried the calibration for "pressure advance" in the creality slicing software and have u done a port into prusa? i tried it and got the value of 0.05-0.07 from my testprint. but till now, i have not ported it to prusa. if ported into prusa or different slicing softwares, the retraction settings should be obsolete and have to be deactivated, as mentioned in the marlin documentary. this could maybe solve the problem of the round edges, but maybe could cause more problems at other places :D.
4. why do u use the z-lift if the filament is retracted? i calibrate every filament with a temp_tower and flow cube, so the z-lift is obsolete in my experience.
5. why do u use "marlin2" as "gcode-type"? i only found that marlin is natively used for the printer. but i'm not quite sure about that. :D 
6. how did u calibrate the retraction settings of the extruder? on my old profile i got quite lower values, while calibrating. but i will do it again with ur settings. 
7. are doing the "auto pid calibration" for nozzle and bed in the firmware for every filament?

edit:
8. whre did ur filament come from? do u use the native spool holder on top of the z-axis? i am using an external drybox made of ikea samla 22l boxes. they build as a tower and i switch the upper box, if i need different filament. so at the place of the spool holder is filament guide to feed the extruder directly from the dry box. the main reason why  i did this is the resonance of the z-axis at high speeds and fast direction changes of the print head - and wet filament. but that only switched the wobble of the z-axis to different speeds and movements and didn't elimate it. whats ur experience with z-axis wobble and how do u feed ur extruder?

so thats it for now. sry for my bad english. if there are problems by undderstanding some parts, pls ask :D.

thanks for ur work my friend!

deadeye

**Comment by suchmememanyskill** [comment]
1. I currently use petg very often, so i just blamed the stringing on petg. It's possible the little bit of z lift is the cause of this, but i haven't had the chance yet to test this.
2. I don't think this is the reason.
3. Pressure Advance/Linear Advance is not available on stock firmware (at least when i tested it, on v1.0.4). Either creailty is faking it with the slicer (making thinner corners) or using a non-standard implementation.
4. Z lift can be quite useful, especially if you print models with thin supports. Not having a tiny bit of Z lift could cause the print head to catch on a support piece and knock it over. That's why i kept it enabled. For petg it's disabled as it causes an unbearable amount of stringing.
5. The Ender 3 V3 SE runs on Marlin v2 (input shaping is not available on v1).
6. I don't, i just increase it if i get bad results. The current default of 0.8 is taken from the S1 prusaslicer profile.
7. No
8. I use a lot of [this manufacturer's PETG/PLA](https://www.123-3d.nl/123-3D-Filament-transparant-zwart-1-75-mm-PETG-1-kg-Jupiter-serie-DFE02012c-DFE02030c-DFP14094c-i1912-t7396.html). I removed the spool holder as it caused quite a bit of vibration. Currently i have a spool holder that holds a spool on a shelf above the 3d printer. I also get Z wobble but have not tried to correct it; i don't think it's obvious enough to care about it.

**Reply by deadeyeflint** [reply]
Thansk a lot for the reply. u mentioned some things that i didnt know.

**Comment by cedbossneo** [comment]
I'm using Klipper via Sonic Pad, do you have a config tuned for klipper ?

**Reply by pwlgrzs** [reply]
It does include klipper config

**Reply by suchmememanyskill** [reply]
It isn't particularly 'tuned well' for Klipper (it doesn't go any faster than the standard marlin profiles), but it does work. I personally use Klipper too with my own profiles.

**Comment by potsu22** [comment]
I still get some pretty funky stringing with these profiles. Messed with temp towers, retractions settings, flow volumes, e step configs , ect. Anyone managed to make significant improvements changing any settings?

**Reply by suchmememanyskill** [reply]
Can you try setting z hop (lift z) to 0 in printer settings? That does decrease stringing by a bit.

Other than that, i genuinely don't know how i would improve it. It's beyond my understanding of slicers & extruders currently

**Reply by potsu22** [reply]
hey just an FYI i messed with settings for ages and had little to no success with stringing. I changed over filament to the stuff that came with it  (the small sample roll) and prints were WAY cleaner. So 3 new boxes of Esun  pla+ caused the issue. I'm trying to dry the stuff now an see if it makes a difference. 

your profile is pretty clean with the standard settings and is a great baseline for future filaments. thanks for your help.
